crates/vm/tests/integration_test.rs

```diff
@@ -56,7 +57,7 @@ where
 
 fn test_native_config() -> NativeConfig {
     NativeConfig {
-        system: SystemConfig::default(),
+        system: SystemConfig::new(3, MemoryConfig::new(2, 1, 16, 29, 15, 32, 1024), 0),
```

Comment:
Good to know this config works now, but I'm not sure it's better than the old one for testing since native address space is 4.

Reply:
4 is still supported right? it exactly hits the edge case: 4 < 1 + 2^2

Reply:
Ah, right.